src/main/webapp/resources/js/components/samples/components/fastqc/FastQCMenu.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Tabs } from "antd";
+
+import { FastQCCharts } from "./FastQCCharts";
+import { OverRepresentedSequences } from "./OverRepresentedSequences";
+import { FastQCDetails } from "./FastQCDetails";
+
+/**
+ * React component to render tab menu for the FastQC results
+ * @returns {JSX.Element}
+ * @constructor
+ */
+export function FastQCMenu() {
+  return (
+    <>
```

Comment:
You love to wrap empty fragments around things :) Not a big deal.

Reply:
It's my favorite thing to do :)